README.md

```diff
@@ -15,89 +15,75 @@ If you haven’t set up a "cloudwatch2scalyr" lambda function yet, both of these
+Specify your API key by doing the following:
 
 * Enter "SCALYR_WRITE_LOGS_KEY" in the first column (no quotes)
 * Provide your account’s actual Scalyr "Write Logs" API Key in the second column.
 * Click "Encrypt".
 
 Once you’ve done this, you should see something like:
 
-<div style="text-align:center"><img src="markdown_images/image_5.png" width="900"></div>
-<br>
+<div style="text-align:center"><img src="markdown_images/image_3.png" width="900"></div>
 
-There are a couple of additional environment variables you can pass to the code - `USE_ADD_EVENTS_API`, `PARSER_NAME`, and `SCALYR_BASE_URL`. (These should not be encrypted.)
+There are additional environment variables you can pass to the code - `USE_ADD_EVENTS_API`, `PARSER_NAME`, and `SCALYR_BASE_URL`. (These should not be encrypted.)
 * In general, `USE_ADD_EVENTS_API` should be false (and you don’t actually need to specify it unless you want it to be true).
 * `PARSER_NAME` is optional as well and refers to a specific custom parser defined using the Scalyr UI - if not specified, the default parser is used.
-* `SCALYR_BASE_URL` defaults to `https://www.scalyr.com` and selects the Scalyr region to which you want to upload your logs, such as  `https://upload.eu.scalyr.com`.  *Note:* If this parameter is configured incorrectly, you will not get error messages from Scalyr during log upload; AWS logs of this Lambda function will look as if everything is working, but the logs won't show in Scalyr.
+* `SCALYR_BASE_URL` defaults to `https://www.scalyr.com` and selects the Scalyr region to which you want to upload your logs, such as  `https://upload.eu.scalyr.com`.  *Note:* If this parameter is configured incorrectly, you will not get error messages from Scalyr during log upload; AWS logs of this Lambda function will look as if everything is working, but the logs won't be displayed in Scalyr.
 
-<div style="text-align:center"><img src="markdown_images/image_6.png" width="600"></div>
+<div style="text-align:center"><img src="markdown_images/image_4.png" width="600"></div>
 <br>
 
-## Define a role for your function
+## Save and test the function
 
-The last step on this page is to define a role for your function. In the next section of the page, choose "Create new role from template(s)" from the Role dropdown, give the role a name, and choose “KMS decryption permissions” from the Policy Templates dropdown:
+After you've entered all necessary information, you can save and test the function. Click on the "Save and test" button in the upper right-hand corner. On thte screen that appears, ensure that "Create new test event" is selected, then select "CloudWatch Logs" from the Event template dropdown. Give the event a name, then click "Create" at the bottom of the page.
 
-<div style="text-align:center"><img src="markdown_images/image_7.png" width="900"></div>
-<br>
-
-## Finalize the installation
-
-Now, click "Next". On the next page, click “Create Function”. You should see a screen like:
```

Comment:
I'd specify that "123456789123" is an example value since someone might look for that exact text...

Reply:
Interesting. 123456789123 is what appears every time I've run it. But yeah, will tweak the doc accordingly.

Reply:
I'd have to write something to decode/decrypt the message generated in the test, I believe -- but that value (cloudwatch-123456789123) comes from the message that AWS generates for testing. Let me know if you think otherwise.